bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,262 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdin=True,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file, csscomb_json: str=""):
+        args = (filename,)
+        if csscomb_json:
+            args += (csscomb_json,)
+        else:
+            args += (config_file,)
```

Comment:
it's kind of weird if the user has to give the config_file and if he gives a second arguent out of a sudden it gets ignored :/